run_test.sh

```diff
@@ -2,7 +2,7 @@
 
 # Run Unit Test for mysql
 
-if [[ $MYSQL_TEST == "true" ]]; then
+if [ $MYSQL_TEST == "true" ] && [ $CIRCLECI != "true" ]; then
```

Comment:
If this script is running on CircleCI env, this env value becomes true. If we run the test on CircleCI, we can get fresh mysql env everytime. Therefore, `drop database` is not required. In this sense, mysql-client is also not required.